pyodide-build/README.md

```diff
@@ -5,6 +5,12 @@ Tools for building Pyodide.
 See [http://github.com/pyodide/pyodide](http://github.com/pyodide/pyodide) for
 more information.
 
+## Upgrading pyodide to a new version of CPython
+For example: `v3.11.1` -> `v3.11.2`
+1. Download the __Gzipped source tarball__ https://www.python.org/downloads/release/python-3112/ into `downloads`
+2. `shasum -a 256 downloads/Python-3.11.2.tgz > cpython/checksums`
+3. `git grep --name-only "3.11.`"  # These files should be updated
+
```

Comment:
Thanks for adding instructions on this. We should maybe add a suggestion to rebase the patches, though this is not needed if they apply cleanly anyways. When updating major versions, there is a step 4 of fix all the broken tests which we should point out can be very complicated.

Also, I think this would most appropriately go in the maintainer docs at docs/development/maintainers.md 
@rth what do you thing?

Reply:
> appropriately go in the maintainer docs

I was going to add the same comment +1. Thanks for working on the update!

Reply:
Moved to `docs/development/maintainers.md`